app/src/main/java/ar/com/wolox/android/example/model/NewData.kt

```diff
@@ -0,0 +1,13 @@
+package ar.com.wolox.android.example.model
+import kotlin.collections.ArrayList
+
+data class NewData(
```

Comment:
We usually try to avoid this kind of wording in data classes. Instead of `NewData`, `NewModel`, `NewResponse`, `NewBody`... we try to simplify it like `New`